.buildkite/packaging.pipeline.yml

```diff
@@ -265,6 +334,76 @@ steps:
         artifact_paths:
           - build/distributions/**/*
 
+      - label: "STAGING: {{matrix}} FIPS"
+        env:
+          PLATFORMS: "${PLATFORMS_FIPS}"
+          SNAPSHOT: false
+          DEV: false
+          FIPS: true
+        command: |
+          source .buildkite/scripts/version_qualifier.sh
+          .buildkite/scripts/packaging/package-dra.sh {{matrix}}
+        agents:
+          provider: gcp
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        timeout_in_minutes: 40
+        retry:
+          automatic:
+            - limit: 1
+        artifact_paths:
+          - build/distributions/**/*
+        matrix:
+          - x-pack/auditbeat
+          - x-pack/filebeat
+          - x-pack/metricbeat
+
+      - label: "STAGING: {{matrix}} Linux/arm64 FIPS"
+        env:
+          PLATFORMS: "${PLATFORMS_ARM_FIPS}"
+          SNAPSHOT: false
+          DEV: false
+          FIPS: true
+        command: |
+          source .buildkite/scripts/version_qualifier.sh
+          .buildkite/scripts/packaging/package-dra.sh {{matrix}}
+        agents:
+          provider: "aws"
+          imagePrefix: "${AWS_IMAGE_UBUNTU_ARM_64}"
+          instanceType: "${AWS_ARM_INSTANCE_TYPE}"
+        timeout_in_minutes: 40
+        retry:
+          automatic:
+            - limit: 1
+        artifact_paths:
+          - build/distributions/**/*
+        matrix:
+          - x-pack/auditbeat
+          - x-pack/filebeat
+          - x-pack/metricbeat
+          - x-pack/agentbeat
+
+        ## Agentbeat needs more CPUs because it builds many other beats
+      - label: "STAGING: x-pack/agentbeat all artifacts apart from linux/arm64 FIPS"
```

Comment:
```suggestion
      - label: "STAGING: x-pack/agentbeat Linux/amd64 FIPS"
```

Reply:
Fixed in e1cf0bd091e854c0a8a936fe6dfb234c0cea98ad